package.json

```
@@ -12,6 +12,7 @@
   ],
   "dependencies": {
     "debug": "4",
+    "driver.js": "^1.3.1",
```

we prefer to use only the major version in package.json as we don't have minimal requirement, it also so that `npm update` only changes the package-lock.json.
```suggestion
    "driver.js": "1",
```